### ISSUE TITLE
Where do parameter definitions go?

### ISSUE THREAD

**Issue by gdsmith** [northwoods/container#13]
I'm unclear where I should put parameter definitions like the following:

```PHP
    $injector->defineParam('globalParam', 'value');

    $injector->define(MyClass::class, [
        ':myParam' => 'value',
    ]);
```

Currently adding them to `config/container.php` but not feeling that it's the "correct" place

**Comment by shadowhand** [comment]
All configuration should go in a [configuration class](https://github.com/northwoods/container#configuration).

**Comment by gdsmith** [comment]
A full example of how to use this with zend-expressive and an example of a configuration class would be helpful

**Comment by shadowhand** [comment]
Fixed in 3.1.0.
